src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -47,6 +47,7 @@ type NotificationTexts = {
 export type OpeningPeriodFormProps = {
   formId: string;
   datePeriod?: DatePeriod;
+  forceException: boolean;
```

Comment:
Could this be optional attribute just for CreatePeriodPage? Something like initialExceptionData or initialFormData and `{ override: true }`?
Then it would not be necessary to use the attribute in the EditPeriodPage

Reply:
This attribute is not required by EditPeriodPage. It is only applied in CreateNewOpeningPeriodPage and OpeningPeriodForm.

But maybe your point is that it's confusing to pass that as a prop for OpeningPeriod form inside the EditPeriodPage? Kind of wouldn't want an abstraction object for only one simple boolean value though.

Reply:
- [ ] Make forceException optional and to default false